lib/console/deployments/observability/webhook.ex

```diff
@@ -54,6 +54,32 @@ defmodule Console.Deployments.Observability.Webhook do
     |> ok()
   end
 
+  def payload(%ObservabilityWebhook{type: :datadog}, payload) do
+    case Datadog.normalize_datadog_payload(payload) do
```

Comment:
normalize_datadog_payload should return an `{:ok, _} | {:error, _}` and be matched against that instead of an empty list or non-empty list, to adhere to more conventional usage.